exercises/raindrops/example.ml

```diff
@@ -0,0 +1,12 @@
+open Core.Std
+
+let raindrop n =
+  match n with
```

Comment:
Just a suggestion for improvement: this file isn't part of the public interface so not a big deal. 

You can use the ```function``` keyword here instead, it's a little shorter. The following blog describes it well. 
http://philogb.github.io/blog/2008/08/11/pattern-matching-in-ocaml-and-javascript/